WalletWasabi.Fluent/Views/Settings/BitcoinTabSettingsView.axaml

```diff
@@ -3,21 +3,20 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              xmlns:settings="using:WalletWasabi.Fluent.ViewModels.Settings"
-             mc:Ignorable="d" d:DesignWidth="428" d:DesignHeight="371"
+             mc:Ignorable="d" d:DesignWidth="650"
              x:DataType="settings:BitcoinTabSettingsViewModel"
              x:Class="WalletWasabi.Fluent.Views.Settings.BitcoinTabSettingsView"
              x:CompileBindings="True">
   <StackPanel Classes="settingsLayout">
 
-    <StackPanel>
+    <DockPanel>
       <TextBlock Text="Network" />
-      <ComboBox HorizontalAlignment="Stretch"
-                ItemsSource="{Binding Networks}"
+      <ComboBox ItemsSource="{Binding Networks}"
                 SelectedItem="{Binding Settings.Network}" />
-    </StackPanel>
+    </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="(EXPERIMENTAL) Run Bitcoin Knots on startup" />
```

Comment:
maybe could drop this change, not sure

Reply:
I wanted to do it for quite some time and used this PR to make the change.

This feature is NOT experimental

![CleanShot 2024-12-04 at 18 01 25@2x](https://github.com/user-attachments/assets/12dbe27a-9a78-4f63-a542-4bf22689c1be)

- It has been tried for a long time
- It is not under active development or will not improve

So we are not doing any kind of experiment here. If the feature is not working we should remove it